central/notifiers/jira/jira.go

```diff
@@ -304,10 +464,23 @@ func newJira(notifier *storage.Notifier, metadataGetter notifiers.MetadataGetter
 	if err != nil {
 		return nil, errors.Wrap(err, "could not create JIRA client")
 	}
-	prios, _, err := client.Priority.GetList()
+
+	// Test auth
+
+	urlPath := "rest/api/2/configuration"
+
+	req, err := client.NewRequest("GET", urlPath, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	log.Debugf("Making request to %s", urlPath)
+	_, err = client.Do(req, nil)
+
 	if err != nil {
 		errStr := err.Error()
-		if strings.HasPrefix(errStr, "401") || strings.HasPrefix(errStr, "403") {
+		if strings.Contains(errStr, "401") || strings.Contains(errStr, "403") {
```

Comment:
I know this is here historically, but does the response code from jira not match the error?

Reply:
updated to check the status code directly from the response struct.